README.md

```diff
@@ -31,6 +31,27 @@ end
 
 NOTE: If you don't already have an environment variable for your Mandrill API key, don't forget to create one.
 
+### Mandrill Templates
+
+If you want to use this gem with mandrill templates you just have to add the `template` param to the `mail` function.
+
+```ruby
+class MyMailer < ActionMailer::Base
+  def notify_user(email)
+    mail(
+      to: email,
+      from: 'your@email.com',
+      template: 'your-mandrill-template-slug',
+      template_content: { # optional
```

Comment:
@genaromadrid we should use Mandrill's syntax for `template_content`, so no one will have to learn one more syntax.
```
    "template_content": [
        {
            "name": "example name",
            "content": "example content"
        }
    ]
```